tests/attr/test_shapley.py

```diff
@@ -333,6 +333,7 @@ def _single_input_multi_sample_batch_scalar_shapley_assert(
             feature_mask=mask,
             perturbations_per_eval=(1,),
             target=None,
+            n_samples = 2500
```

Comment:
Looks like there's a flake8 line issue in the CircleCI results, would you be able to fix this?